.bit/responses/4.4-Week 4 Step 4.md

```diff
@@ -1,16 +1,27 @@
 ---
-files: 
-stepType: 
+files: n/a
+stepType: checks
 scripts: 
 week: 4
 step: 4
 name: Week 4 Step 4
 ---
 
-## Week 4 Step 4
+Week 4 Step 4 ⬤⬤⬤⬤◯◯◯◯◯ | 🕐 Estimated completion: 5-20 minutes
 
-### The Lightning Talk
+## Downloading your image!
 
-Your final step is to prepare a 15 minute talk and demo of your project. Describe your vision, your process, and any challenges that occured. Use slides to enhance your talk through images, gifs, and perhaps short code snippets - *not* paragraphs of text.
+Write and connect a JS file to make a GET request to the download Azure function that includes the codename to receive the link.
+
+Edit the button on the HTML page so that it downloads the file when clicked. 
+
+### ✅  Task:
+- [ ] Create a new input field called `downloadusername` and create a button that calls the function `downloadImage()` in `index.html` 
+- [ ] Create the `downloadImage()` function with a `await fetch` GET request to send `downloadusername` to your Bunnimage Azure Function to return the file download uri
+- [ ] If the fetch was successful, open the file download link using [`window.open()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/open)

```

Comment:
Add a section explaining window.open() and it's parameters (and link docs)
Specify that it must open in the same tab